pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/LLCRealtimeClusterIntegrationTest.java

```diff
@@ -36,18 +38,22 @@
  */
 public class LLCRealtimeClusterIntegrationTest extends RealtimeClusterIntegrationTest {
   private static int KAFKA_PARTITION_COUNT = 2;
+  private static final String KAFKA_PARTITIONING_KEY = "AirlineID";
 
   protected void setUpTable(String tableName, String timeColumnName, String timeColumnType, String kafkaZkUrl,
       String kafkaTopic, File schemaFile, File avroFile) throws Exception {
     Schema schema = Schema.fromFile(schemaFile);
     addSchema(schemaFile, schema.getSchemaName());
+    Map<String, String> map = new HashMap<String, String>();
+    map.put(KAFKA_PARTITIONING_KEY, PartitionFunctionFactory.PartitionFunctionType.ByteArray.toString());
```

Comment:
With this change, we are always testing with partition key?
Can we please add a separate test for testing partition key so that we still have other tests that do not use partition key but still use llc?